.github/workflows/deploy.yml

```diff
@@ -45,10 +45,13 @@ jobs:
           path: "ocw-hugo-projects"
 
       - name: Run webpack build
-        run: npm run build:webpack -- --output-path=../ocw-www/public/static/
+        run: npm run build:webpack -- --output-path=../ocw-www/public/static_shared/
 
+      - name: Copy webpack build to course
+        run: mkdir -p ../ocw-course/public-v1/ && cp -r ../ocw-www/public/static_shared/ ../ocw-course/public-v1/
```

Comment:
**Requested Change:** This should be gone now after #1051 , right?

Reply:
Yep, must have been a bad rebase. Good eye!